client/Bootstrap/index.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import 'Styling';
+import image from 'Images/logo.png';
+
+window.addEventListener('load', () => {
+    const root = document.querySelector('[id="root"]');
```

Comment:
I'm guessing this is just so that we have placeholder JS to prove webpack is bundling? And will later replace with React logic?

Reply:
As per the PR, yes, completley placeholder/Hello world, to use the build. All bar fonts have been utilised in this UI